src/components/User/index.js

```diff
@@ -9,18 +9,33 @@ export default class User extends Component {
   state = {
     loading: true
   };
+
+  handleError(err) {
+    let errorMessage = "There is something wrong, open log for more information!";
+    if (err.response &&
+        err.response.data &&
+        err.response.data.errors &&
+        err.response.data.errors.length > 0
+    ) {
+      errorMessage = err.response.data.errors.map(e => ("- " + e.message)).join("\r\n");
```

Comment:
Codacy is complaining about this line

Reply:
My bad, misread the error before. That should do it